### DISCUSSION TITLE
Reuse some of the Code in a new Kibana's visualisation

### DISCUSSION THREAD

**Discussion by ch-bas** [fbaligand/kibana-enhanced-table#183] [General]
Hi,
I'm trying to update my plugin (visualisation) to be compatible with Kibana's new platform, but as far as their is no official way how to do it, I'm asking is it possible to use part of your code in my plugin ( especially the part of importing the legacy Kibana's code)?  
Thanks for your response!

**Comment by fbaligand** [comment]
Hi,

enhanced table code is under Apache 2.0 open source license.
it means that you can do whatever you want.
This includes copy/paste some parts of enhanced table plugin code  into your plugin.
